src/store/getters.ts

```diff
@@ -77,5 +77,14 @@ export default {
       }
     }
     return lastFilingDate
+  },
+
+  isBusinessWithNoMaintenanceFilings (state): boolean {
+    if (state.filings && state.filings.length === 1 &&
```

Comment:
Please add a comment explaining why this code is needed / how it works. I guess, something like "this is to show the Legal Obligations only for a new business that hasn't filing anything else".

BTW, if they have an Annual Report in their Todo List, does this display? Seems a bit weird but I dunno.

Reply:
Sure will do

Reply:
Ya, It doesnt make sense to show this if there is something in the todo list. I will add that condition